python_on_whales/components/buildx/cli_wrapper.py

```diff
@@ -81,6 +84,18 @@ def remove(self):
         """
         BuildxCLI(self.client_config).remove(self)
 
+    @property
+    def implementation(self) -> str:
```

Comment:
I believe this information should go in the https://github.com/gabrieldemarmiesse/python-on-whales/blob/master/python_on_whales/client_config.py#L45

The api should look like this
```python
from python_on_whales import DockerClient
podman = DockerClient(client_binary="podman")
output = podman.run("hello-world")
print(output)
```

In this way, that would also work with nerdctl for example.

You can also modify https://github.com/gabrieldemarmiesse/python-on-whales/blob/master/python_on_whales/client_config.py#L64 so that it will search for the "client_binary" instead of docker